conda_build/build.py

```diff
@@ -458,15 +458,19 @@ def test(m, verbose=True, channel_urls=(), override_channels=False):
     rm_rf(config.test_prefix)
     specs = ['%s %s %s' % (m.name(), m.version(), m.build_id())]
 
-    if py_files:
+    # add packages listed in test/requires
+    specs_include_python = False
+    for spec in m.get_value('test/requires', []):
+        specs.append(spec)
```

Comment:
@asmeurer They were just moved here, taken care of before the `py_files` check. I just think the diff is not picking up on the move.